drivers/gles3/rasterizer_gles3.cpp

```diff
@@ -210,6 +210,9 @@ RasterizerGLES3::RasterizerGLES3() {
 #ifdef GLAD_ENABLED
 	if (!gladLoadGL()) {
 		ERR_PRINT("Error initializing GLAD");
+		// FIXME this is an early return from a constructor.  Any other code using this instance will crash or the finalizer will crash, because none of
+		// the members of this instance are initialized, so this just makes debugging harder.  It should either crash here intentionally,
+		// or we need to actually test for this situation before constructing this.
```

Comment:
In `3.x` this is done in `RasterizerGLES3::is_viable()`. Something similar could likely be done here.

Reply:
I think this is probably a good idea.  I am worried that your comment will just die here because nobody is going to come back to the PR later.   Should we put that in the code then?   For the record, I am working closely with the current main author of that code, so he is going to fix this (or I will) very shortly anyway, so this is probably academic.

Reply:
Yeah then the comment you added is probably fine as is :+1: